code/game/antagonist/station/changeling.dm

```diff
@@ -1,11 +1,16 @@
-/datum/antagonist/changeling
-	id = ROLE_CHANGELING
-	role_text = "Changeling"
-	role_text_plural = "Changelings"
+/datum/antagonist/carrion
+	id = ROLE_CARRION
+	role_text = "Carrion"
+	role_text_plural = "Carrions"
 	restricted_jobs = list("AI", "Robot")
 	protected_jobs = list(JOBS_SECURITY, JOBS_COMMAND)
-	bantype = ROLE_BANTYPE_CHANGELING
-	welcome_text = "Use say \"#g message\" to communicate with your fellow changelings. Remember: you get all of their absorbed DNA if you absorb them."
+	bantype = ROLE_BANTYPE_CARRION
+	welcome_text = "You are Carrion, a leftover from corporate war. You have come onto this vessel to carry out your master plan, and no one can stop you.<br>\
+	Your body is ever changing. You should start out by evolving a chemical vessel to use your powers. A carrion maw can be a good way to earn evolution points fast. <br>\
+	You can do contracts to grow stronger until the ship becomes your stage and your master plan is ready. A slow and methodical approach is recommended. <br>\
+	You won't find many friends here, but spiders are one of them. If you ever feel alone, you can always make more friends. <br>\
```

Comment:
```suggestion
	You won't find many friends here, but spiders are one of them. If you ever feel alone, you can always give birth to your own children, or search ship in attempt to find your brothers and sisters. <br>\
```
Maybe it will be better.